src/changelog/unreleased.md

```diff
@@ -101,8 +104,12 @@ changelog entry.
   This feature was incomplete, and the equivalent functionality can be trivially achieved outside
   of `winit` using `objc2-ui-kit` and calling `UIDevice::currentDevice().userInterfaceIdiom()`.
 - On Web, remove unused `platform::web::CustomCursorError::Animation`.
+- Remove `MonitorHandle::size()` and `refresh_rate_millihertz()` in favor of
+  `MonitorHandle::current_video_mode()`.
+- On Android, remove all `MonitorHandle` support instead of emitting false data.
```

Comment:
"and `VideoModeHandle`", or is that implicit to `MonitorHandle`?

Reply:
Yes, because without a `MonitorHandle` there is no way to acquire a `VideoModeHandle`.